src/v3/src/components/PasswordRequirements/PasswordRequirements.tsx

```diff
@@ -39,22 +37,25 @@ const PasswordRequirements: UISchemaElementComponent<{
 }> = ({ uischema }) => {
   const { data } = useWidgetContext();
   const {
-    fieldKey,
     header,
     id,
     requirements,
     settings,
     userInfo,
     validationDelayMs,
   } = uischema.options as PasswordRequirementsElement['options'];
-  const password = get(data, fieldKey);
+  // TODO: OKTA-521321 - Depending on the outcome of this ticket, we may be able to remove this
+  // If it is kept, need to add integration test for validation
+  const password = 'credentials.newPassword' in data
```

Comment:
curious can you get password here during debugging? data should be an empty object when reach here

Reply:
100% can, this is how the icons on the password requirements currently updates (changes from ❌  to ✅ ) using this same thing, the only difference here is I stopped passing this key from transformer.

Reply:
ohh, this one is under components dir, missed that